### PR TITLE
Revise tutorial to separate out platforms

### PR DESCRIPTION
**Pull request by natke**

### CODE REVIEW

**Review by baijumeswani — APPROVED** [review]

**Review by parinitarahi — COMMENTED** [review]
The `-m` argument is the path to the model you downloaded from HuggingFace above. ---> should be The `-m` argument is path to the model folder downloaded from HuggingFace above.

File: examples/python/phi-3-tutorial.md [hunk 1]
```diff
@@ -1,93 +1,149 @@
...
+   ```bash
+   pip install huggingface-hub[cli]
+   ```
 
-Are you on Windows machine with GPU?
+## Choose your platform
+
+Are you on a Windows machine with GPU?
 * I don't know &rarr; Review [this guide](https://www.microsoft.com/en-us/windows/learning-center/how-to-check-gpu) to see whether you have a GPU in your Windows machine.
 * Yes &rarr; Follow the instructions for [DirectML](#directml).
```
Comment (sophies927) [inline examples/python/phi-3-tutorial.md:40]:
```suggestion
* Yes &rarr; Follow the instructions for [DirectML](#run-with-directml).
```

File: examples/python/phi-3-tutorial.md [hunk 2]
```diff
@@ -1,93 +1,149 @@
...
 * No &rarr; Do you have an NVIDIA GPU?
   * I don't know &rarr; Review [this guide](https://docs.nvidia.com/cuda/cuda-installation-guide-microsoft-windows/index.html#verify-you-have-a-cuda-capable-gpu) to see whether you have a CUDA-capable GPU.
   * Yes &rarr; Follow the instructions for [NVIDIA CUDA GPU](#nvidia-cuda-gpu).
```
Comment (sophies927) [inline examples/python/phi-3-tutorial.md:43]:
```suggestion
  * Yes &rarr; Follow the instructions for [NVIDIA CUDA GPU](#run-with-nvidia-cuda).
```

File: examples/python/phi-3-tutorial.md [hunk 3]
```diff
@@ -1,93 +1,149 @@
...
   * No &rarr; Follow the instructions for [CPU](#cpu).
```
Comment (sophies927) [inline examples/python/phi-3-tutorial.md:44]:
```suggestion
  * No &rarr; Follow the instructions for [CPU](#run-on-cpu).
```

**Review by sophies927 — COMMENTED** [review]
Made some suggestions to fix links in "Choose your platform" section

**Comment by natke** [comment]
> The `-m` argument is the path to the model you downloaded from HuggingFace above. ---> should be The `-m` argument is path to the model folder downloaded from HuggingFace above.

That's no longer in the current tutorial. Do you want me to add it?

File: examples/python/phi-3-tutorial.md (hunk 1 above)
Comment (natke) [reply examples/python/phi-3-tutorial.md]: Fixed

File: examples/python/phi-3-tutorial.md (hunk 2 above)
Comment (natke) [reply examples/python/phi-3-tutorial.md]: Fixed

File: examples/python/phi-3-tutorial.md (hunk 3 above)
Comment (natke) [reply examples/python/phi-3-tutorial.md]: Fixed

**Pull request merged**